src/Exception/MessageBagNotFound.php

```diff
@@ -6,7 +6,7 @@
 
 final class MessageBagNotFound extends \DomainException
```

Comment:
Is this class used somewhere?

Reply:
no, good catch, the caller is gone since a while.

it was an early MessageStore implementation:
https://github.com/php-llm/llm-chain/blob/4804cf0fc75130a8eb76e03c77cb4a6c6a4cb4cc/src/Message/MessageStore.php